<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_sec_sts">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>750</y>
      <width>521</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection status</string>
    </property>
   </widget>
   <widget class="QTableView" name="table_event">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>661</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView
{border-radius:10px;
background-color:rgb(100,100,100);
color:rgb(206, 206, 206);}
QHeaderView::section {
    background-color: rgb(80, 80, 80);
    color: white;
	height: 35px;
    font: 14px;
}</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>781</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color:rgb(248, 246, 240);
border:1px solid black;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton{
background-color: rgb(93, 93, 93);
color:white ;
border-radius:10px;
}
*/
QPushButton
{
background-color: rgb(93, 93, 93);
color:black ;
border-radius:10px;
border-bottom:3px solid #aaaaaa;
}
QPushButton:enabled
{
background-color: rgb(230, 180, 200)  ; 
border-color:black;
}

QPushButton:pressed
{
background-color: rgb(230, 150, 180)   ;
color:  black   ;
}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_verifier">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton{
background-color: rgb(93, 93, 93);
color:white ;
border-radius:10px;
}
*/
QPushButton
{
background-color: rgb(93, 93, 93);
color:black ;
border-radius:10px;
border-bottom:3px solid #aaaaaa;
}
QPushButton:enabled
{
background-color: rgb(230, 180, 200)  ; 
border-color:black;
}

QPushButton:pressed
{
background-color: rgb(230, 150, 180)   ;
color:  black   ;
}</string>
     </property>
     <property name="text">
      <string>verfier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modifier">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton{
background-color: rgb(93, 93, 93);
color:white ;
border-radius:10px;
}
*/
QPushButton
{
background-color: rgb(93, 93, 93);
color:black ;
border-radius:10px;
border-bottom:3px solid #aaaaaa;
}
QPushButton:enabled
{
background-color: rgb(230, 180, 200)  ; 
border-color:black;
}

QPushButton:pressed
{
background-color: rgb(230, 150, 180)   ;
color:  black   ;
}</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton{
background-color: rgb(93, 93, 93);
color:white ;
border-radius:10px;
}
*/
QPushButton
{
background-color: rgb(93, 93, 93);
color:black ;
border-radius:10px;
border-bottom:3px solid #aaaaaa;
}
QPushButton:enabled
{
background-color: rgb(230, 180, 200)  ; 
border-color:black;
}

QPushButton:pressed
{
background-color: rgb(230, 150, 180)   ;
color:  black   ;
}</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border:2px,solid white;
border-radius:10px;
color:black;
padding-left :10px;
padding-right:20px;
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover
{
border:1.5px solid rgb(48,50,62);
}
QLineEdit
{
border:1.5px solid rgb(255, 170, 255);
background-color: white;
}
QLineEdit:focus
{
border:1.5px solid rgb(48,50,62);
}</string>
     </property>
     <property name="placeholderText">
      <string>  ID a supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_budget_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>-40</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_chercher">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton{
background-color: rgb(93, 93, 93);
color:white ;
border-radius:10px;
}
*/
QPushButton
{
background-color: rgb(93, 93, 93);
color:black ;
border-radius:10px;
border-bottom:3px solid #aaaaaa;

}
QPushButton:enabled
{
background-color: rgb(230, 180, 200)  ; 
border-color:black;
}

QPushButton:pressed
{
background-color: rgb(230, 150, 180)   ;
color:     black   ;
}</string>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_chercher">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border:2px,solid white;
border-radius:10px;
color:black;
padding-left :10px;
padding-right:20px;
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover
{
border:1.5px solid rgb(48,50,62);
}
QLineEdit
{
border:1.5px solid rgb(255, 170, 255);
background-color: white;
}
QLineEdit:focus
{
border:1.5px solid rgb(48,50,62);
}</string>
     </property>
     <property name="placeholderText">
      <string>Element a chercher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_chercher_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton{
background-color: rgb(93, 93, 93);
color:white ;
border-radius:10px;
}
*/
QPushButton
{
background-color: rgb(93, 93, 93);
color:black ;
border-radius:10px;
border-bottom:3px solid #aaaaaa;
}
QPushButton:enabled
{
background-color: rgb(230, 180, 200)  ; 
border-color:black;
}

QPushButton:pressed
{
background-color: rgb(230, 150, 180)   ;
color:  black   ;
}</string>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_reminder">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton{
background-color: rgb(93, 93, 93);
color:white ;
border-radius:10px;
}
*/
QPushButton
{
background-color: rgb(93, 93, 93);
color:black ;
border-radius:10px;
border-bottom:3px solid #aaaaaa;
}
QPushButton:enabled
{
background-color: rgb(230, 180, 200)  ; 
border-color:black;
}

QPushButton:pressed
{
background-color: rgb(230, 150, 180)   ;
color:  black   ;
}</string>
     </property>
     <property name="text">
      <string>reminder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton{
background-color: rgb(93, 93, 93);
color:white ;
border-radius:10px;
}
*/
QPushButton
{
background-color: rgb(93, 93, 93);
color:black ;
border-radius:10px;
border-bottom:3px solid #aaaaaa;
}
QPushButton:enabled
{
background-color: rgb(230, 180, 200)  ; 
border-color:black;
}

QPushButton:pressed
{
background-color: rgb(230, 150, 180)   ;
color:  black   ;
}</string>
     </property>
     <property name="text">
      <string>stats</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color:rgb(248, 246, 240);
border:1px solid black;

</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="le_budget">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>230</y>
       <width>301</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border:2px,solid white;
border-radius:10px;
color:black;
padding-left :10px;
padding-right:20px;
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover
{
border:1.5px solid rgb(48,50,62);
}
QLineEdit
{
border:1.5px solid rgb(255, 170, 255);
background-color: white;
}
QLineEdit:focus
{
border:1.5px solid rgb(48,50,62);
}</string>
     </property>
     <property name="placeholderText">
      <string>  Bughet de l'evenement</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>301</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border:2px,solid white;
border-radius:10px;
color:black;
padding-left :10px;
padding-right:20px;
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover
{
border:1.5px solid rgb(48,50,62);
}
QLineEdit
{
border:1.5px solid rgb(255, 170, 255);
background-color: white;
}
QLineEdit:focus
{
border:1.5px solid rgb(48,50,62);
}</string>
     </property>
     <property name="placeholderText">
      <string> Identifiant</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nombre">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>180</y>
       <width>301</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border:2px,solid white;
border-radius:10px;
color:black;
padding-left :10px;
padding-right:20px;
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover
{
border:1.5px solid rgb(48,50,62);
}
QLineEdit
{
border:1.5px solid rgb(255, 170, 255);
background-color: white;
}
QLineEdit:focus
{
border:1.5px solid rgb(48,50,62);
}</string>
     </property>
     <property name="placeholderText">
      <string>  Nombre des invites</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>301</width>
       <height>26</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">

QComboBox
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #111;
    selection-color: black;
    color: white;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 10;
    padding: 1px 0px 1px 20px;
}


QComboBox:hover, QPushButton:hover
{
    border: 1px solid yellow;
    color: white;
}

QComboBox:editable {
    background: red;
    color: pink;
}

QComboBox:on
{
    padding-top: 0px;
    padding-left: 0px;
    color: white;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
    selection-background-color: #ffaa00;
}

QComboBox:!on
{
    color: white;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #666, stop: 0.1 #555, stop: 0.5 #555, stop: 0.9 #444, stop: 1 #333);
}

QComboBox QAbstractItemView
{
    border: 2px solid darkgray;
    color: black;
    selection-background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #111, stop: 1 #333);
}

QComboBox::drop-down
{
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;
     color: white;
     border-left-width: 0px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
     padding-left: 10px;
 }

QComboBox::down-arrow, QSpinBox::down-arrow, QTimeEdit::down-arrow, QDateEdit::down-arrow
{
     image: url(:/icons/down_arrow.png);
     width: 7px;
     height: 5px;
}</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>Anniversaire</string>
     </property>
     <item>
      <property name="text">
       <string>Anniversaire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mariage</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fiancaille</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_ID">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>470</y>
       <width>301</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>250</width>
       <height>250</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
border:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="l_id">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="l_budget">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>260</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="l_nombre">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>210</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="l_date">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>160</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="l_type">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>110</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="themebox">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>10</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:black;
color:white;</string>
     </property>
     <item>
      <property name="text">
       <string>Light Mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dark Mode</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="langue">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>30</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:black;
color:white;</string>
     </property>
     <item>
      <property name="text">
       <string>Francais</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
    </widget>
    <widget class="QDateEdit" name="dateedit">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>130</y>
       <width>301</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
border-color:black;
border-radius:10px;
color:black;
</string>
     </property>
     <property name="displayFormat">
      <string>      dd  /  MM  /  yyyy</string>
     </property>
    </widget>
    <zorder>le_budget</zorder>
    <zorder>le_id</zorder>
    <zorder>le_nombre</zorder>
    <zorder>comboBox</zorder>
    <zorder>label</zorder>
    <zorder>txt_ID</zorder>
    <zorder>l_budget</zorder>
    <zorder>l_id</zorder>
    <zorder>l_type</zorder>
    <zorder>l_date</zorder>
    <zorder>l_nombre</zorder>
    <zorder>themebox</zorder>
    <zorder>langue</zorder>
    <zorder>dateedit</zorder>
    <zorder>label_2</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>781</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color:rgb(248, 246, 240);
border:1px solid black;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pb_trie_id">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton{
background-color: rgb(93, 93, 93);
color:white ;
border-radius:10px;
}
*/
QPushButton
{
background-color: rgb(93, 93, 93);
color:black ;
border-radius:10px;
border-bottom:3px solid #aaaaaa;
}
QPushButton:enabled
{
background-color: rgb(230, 180, 200)  ; 
border-color:black;
}

QPushButton:pressed
{
background-color: rgb(230, 150, 180)   ;
color:  black   ;
}</string>
     </property>
     <property name="text">
      <string>Trie par ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton{
background-color: rgb(93, 93, 93);
color:white ;
border-radius:10px;
}
*/
QPushButton
{
background-color: rgb(93, 93, 93);
color:black ;
border-radius:10px;
border-bottom:3px solid #aaaaaa;
}
QPushButton:enabled
{
background-color: rgb(230, 180, 200)  ; 
border-color:black;
}

QPushButton:pressed
{
background-color: rgb(230, 150, 180)   ;
color:  black   ;
}</string>
     </property>
     <property name="text">
      <string>Trie par nbr_inv</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton{
background-color: rgb(93, 93, 93);
color:white ;
border-radius:10px;
}
*/
QPushButton
{
background-color: rgb(93, 93, 93);
color:black ;
border-radius:10px;
border-bottom:3px solid #aaaaaa;
}
QPushButton:enabled
{
background-color: rgb(230, 180, 200)  ; 
border-color:black;
}

QPushButton:pressed
{
background-color: rgb(230, 150, 180)   ;
color:  black   ;
}</string>
     </property>
     <property name="text">
      <string>pdf</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="khalfeya">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <zorder>khalfeya</zorder>
   <zorder>label_sec_sts</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>table_event</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
